Open_Acidification_pH-stat_arduino/HandleConfig.ino

```diff
@@ -8,16 +8,16 @@ void HandleConfig(char* body, EthernetClient client) {
   } else {
     PrintHeader(client, 200);
     granularity = doc["granularity"];
-    maxDataAge = doc["maxDataAge"];
+    max_data_age = doc["max_data_age"];
```

Comment:
At this point, lets not change string keys inside an array. Those I think may be used to create the JSON which is read by the server. This would mean changing the server code. Lets keep the changes to only the arduino code.